middleware/middleware-1.3.30.md

```diff
@@ -0,0 +1,40 @@
+---
+slug: /release-notes/middleware/1.3.30
+title: Version 1.3.30
+---
+
+# fiskaltrust.Middleware 1.3.30 (Germany)
+_March 17, 2022_
+
+In this version of the Middleware, we've added the possibility to declare a 'Master Queue' if there are multiple Queues connected to one TSE, in order to avoid long timeouts during daily closings and have one tar file export for all queues.
+
+:::caution
+
+Version 1.3 of the Middleware is meant for the German market only, customers in Austria and France should continue to use version 1.2. We will unify these experiences in an upcoming version.
+
+:::
+
+## Feature: Possibility to enable a 'Master Queue' for TAR file export
+
+When setting up the parameter `EnableTarFileExport`, there is a possibility to select one queue which will be responsible for the daily closing and Tar file export.
```

Comment:
Maybe add a second, more detailled hint: "This can be achieved by setting the new parameter to `false` for Queues non-master Queues.